packages/eas-cli/src/commandUtils/flags.ts

```diff
@@ -10,6 +20,42 @@ export const EasNonInteractiveAndJsonFlags = {
   }),
 };
 
+export const EASEnvironmentFlag = {
+  environment: Flags.enum({
+    description: 'Environment to create the secret in',
+    parse: upperCaseAsync,
+    options: mapToLowercase([
+      EnvironmentVariableEnvironment.Development,
+      EnvironmentVariableEnvironment.Preview,
+      EnvironmentVariableEnvironment.Production,
+    ]),
+  }),
+};
+
+export const EASVariableFormatFlag = {
+  format: Flags.string({
+    description: 'Output format',
+    options: ['long', 'short'],
+    default: 'short',
+  }),
+};
+
+export const EASVariableSensitiveFlag = {
+  sensitive: Flags.boolean({
+    description: 'Encrypt variable value at rest',
+    default: false,
+  }),
+};
```

Comment:
how can one create a secret variable? Maybe this flag should be `--type public|sensitive|secret`?

Reply:
I added `--secret` and `--public` flags.